### ISSUE TITLE
Cannot update or reinstall 

### ISSUE THREAD

**Issue by Mosher23** [marcolivierarsenault/moonraker-home-assistant#286]
## Version of the moonraker integration

1.1.0 update to 1.1.1

## Describe your setup

irrelevant

## Describe the bug
when trying to update to 1.1.1 getting error "Empty filename"

I have also tried uninstalling it and reinstalling via HACS again, but installation never succeed

## Moonraker Logs

```
Logger: homeassistant.components.websocket_api.http.connection
Source: components/websocket_api/commands.py:240
Integration: Home Assistant WebSocket API (documentation, issues)
First occurred: 20:30:01 (1 occurrences)
Last logged: 20:30:01

[140469872307648] Empty filename.
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/components/websocket_api/commands.py", line 240, in handle_call_service
    response = await hass.services.async_call(
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/core.py", line 2279, in async_call
    response_data = await coro
                    ^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/core.py", line 2316, in _execute_service
    return await target(service_call)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/helpers/service.py", line 892, in entity_service_call
    single_response = await _handle_entity_call(
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/helpers/service.py", line 962, in _handle_entity_call
    result = await task
             ^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/components/update/__init__.py", line 161, in async_install
    await entity.async_install_with_progress(version, backup)
  File "/usr/src/homeassistant/homeassistant/components/update/__init__.py", line 465, in async_install_with_progress
    await self.async_install(version, backup)
  File "/config/custom_components/hacs/update.py", line 85, in async_install
    await self.repository.async_install()
  File "/config/custom_components/hacs/repositories/base.py", line 899, in async_install
    await self.async_install_repository()
  File "/config/custom_components/hacs/repositories/base.py", line 982, in async_install_repository
    await self.download_content()
  File "/config/custom_components/hacs/repositories/base.py", line 629, in download_content
    await self.download_repository_zip()
  File "/config/custom_components/hacs/repositories/base.py", line 686, in download_repository_zip
    zip_file.extractall(self.content.path.local, extractable)
  File "/usr/local/lib/python3.12/zipfile/__init__.py", line 1721, in extractall
    self._extract_member(zipinfo, path, pwd)
  File "/usr/local/lib/python3.12/zipfile/__init__.py", line 1762, in _extract_member
    raise ValueError("Empty filename.")
ValueError: Empty filename.
```

**Comment by SmarthomeTobi** [comment]
Same Problem here...

**Comment by Mosher23** [comment]
Fixed by updating HACS to version 1.34.0
